conflux/test/adapter.test.ts

```diff
@@ -0,0 +1,90 @@
+import { assert } from 'chai'
+import { Requester } from '@chainlink/external-adapter'
+import { assertSuccess, assertError } from '@chainlink/adapter-test-helpers'
+import { AdapterRequest } from '@chainlink/types'
+import { makeExecute } from '../src/adapter'
+
+describe('execute', () => {
+  const jobID = '1'
+  const execute = makeExecute()
```

Comment:
Then when making the execute, you can pass your own config, setting your own rpcUrl, networkId and PK for testing